mockgen/mockgen.go

```diff
@@ -622,22 +628,28 @@ func (g *generator) Output() []byte {
 	return src
 }
 
-func lookupPackageName(importPath string) (string, bool) {
+// createPackageMap returns a map of import path to package name
+// for specified importPaths.
+func createPackageMap(importPaths []string) map[string]string {
 	var pkg struct {
-		Name string
+		Name       string
+		ImportPath string
 	}
+	names := make(map[string]string)
```

Comment:
Lets rename this to something like `pkgMap`

Reply:
renamed to pkgMap